Makefile

```diff
@@ -17,6 +17,15 @@ integration-tests:
 	fi
 	@./integration-tmp/bin/bats ./test/integration/dnet
 
+./cmd/dnet/dnet:
+	make build
```

Comment:
That did not work on my machine, which is why I decided to use the build target.
Let me investigate more why it is failing.